docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1785,7 +1793,7 @@ If `name` is a specific name and tag (e.g. ubuntu:latest), then only that image
 image (and its parents) are returned, but with the exclusion of the
 'repositories' file in the tarball, as there were no image names referenced.
 
-See the [image tarball format](#image-tarball-format) for more details.
+See the [image tarball format](docker_remote_api_v1.18.md#image-tarball-format) for more details.
```

Comment:
We need the file portion to  be in the link even if it is redundant, because of the way Jekyll handles relative links. @thaJeztah FYI